messages.go

```diff
@@ -178,9 +248,12 @@ type ApplicationData struct {
 	Data []byte `tls:"head=4"`
 }
 
+type Confirmation struct {
+	Data []byte `tls:"head=1"`
+}
```

Comment:
Is there a reason you're isolating this in its own struct?  I sort of understood it for Signature, since that appears multiple places.  But this is a one-off.

Reply:
I see, for use with `Stream.WriteAll`